content/tutorials/exercises/real-challenges.md

```diff
@@ -0,0 +1,56 @@
++++
+title = "Real challenges"
+weight = 4
++++
+
+{% question(title="Challenge") %}
+Print the last 5 most used words from a book.
```

Comment:
Print the five most used words from a book. For better results it is useful to remove the most common words in the language the book was written in (thees are called stopwords).